### DISCUSSION TITLE
How to display custom activity message? is it possible?

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#15193] [Feature Requests]
How to display custom activity message? is it possible?

![image](https://user-images.githubusercontent.com/36895131/185763144-4d5bf0ee-16ee-4df2-9530-f76c336e7706.png)

I want to display a custom activity message once user click Manual Flow.

**Comment by azrikahar** [comment]
It is not currently possible. I believe this is the same use case as #15026 and #14201, so feel free to add your upvote 👍

**Reply by rijkvanzanten** [reply]
It's _kinda_ possible. Custom app extensions can access the NotificationsStore and push additional notifications

**Reply by Jeyakumarrathnaselvi** [reply]
Ooh Great.can you please explain how

**Reply by Jeyakumarrathnaselvi** [reply]
https://docs.directus.io/extensions/displays.html#accessing-internal-systems

you mean using such code. I am not a coder, Can someone guide how to build custom extension to access the NotificationsStore and push additional notifications 

```js
import { useApi, useStores } from '@directus/extensions-sdk';

export default {
	setup() {
		const api = useApi();

		const { useCollectionsStore } = useStores();
		const collectionsStore = useCollectionsStore();

		// ...
	},
};
```

**Reply by rijkvanzanten** [reply]
In this particular case, there isn't really an extension type that can be used to send a notification item like that when a flow completes. In this case, the best way forwards is to create a new feature request to allow the user to configure a custom message based on the flow data when the manual flow completes :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
